src/input/mod.rs

```diff
@@ -2048,10 +2056,28 @@ impl State {
                 }
             }
             Action::ToggleTiling => {
-                let output = seat.active_output();
-                let workspace = self.common.shell.workspaces.active_mut(&output);
-                let mut guard = self.common.shell.workspace_state.update();
-                workspace.toggle_tiling(seat, &mut guard);
+                if self.common.config.config.tile_all_windows {
+                    let autotile = !self.common.config.config.autotile;
```

Comment:
So the toggle tiling shortcut can now enabled/disables tiling globally? Is that dependant of the applet enabling "all workspaces" vs "per workspace"?

Reply:
Yes, as far as I've understood the design, and the text for the autotile toggle changes from "Automatically tile window" to "Automatically tile current workspace" when changing from "All workspaces" to "Per workspace". (I think the text may have a typo and it should probably say "Automatically tile all windows")

Reply:
AH with that typo explained, that name makes sense now. So we are using cosmic-comp for all the settings regarding the applet. Makes perfect sense to store this then, but I dislike the name for that setting as it was highly confusing to me. :sweat_smile:

This might be a bit too verbose, but could we name the key: `tiling_toggle` and not have a `bool` but something like: `enum ToggleBehaviour { Globally, PerWorkspace }` as the value?

Reply:
Ya, I'll double check I guess that my assumption was correct about the typo :sweat_smile: but I'm ok changing it to an enum